stream_alert/rule_processor/classifier.py

```diff
@@ -142,19 +130,20 @@ def map_source(self, payload):
                 break
 
         # if the payload's entity is found in the config and contains logs
-        if self._payload_logs(payload):
-            payload.valid_source = True
+        self._entity_log_sources = self._payload_logs(payload)
+
+        return self._entity_log_sources and len(self._entity_log_sources) > 0
```

Comment:
The first half isn't necessary either. @ryandeivert pointed out that we can just do:
```python
return bool(self._entity_log_sources)
```